docs/submitting-applications.md

```diff
@@ -146,7 +146,7 @@ export HADOOP_CONF_DIR=XXX
 ./bin/spark-submit \
   --class org.apache.spark.examples.SparkPi \
   --master k8s://xx.yy.zz.ww:443 \
-  --deploy-mode cluster \
+  --deploy-mode cluster \  # can be client for client mode
```

Comment:
Is it ture for L145 `#Run on a Kubernetes cluster in cluster deploy mode`

Reply:
@huskysun . Please revert this change. 
As @yaooqinn mentioned, this is irrelevant. In addition, you cannot add `#` after `\` in `bash`, can you?

Reply:
@yaooqinn @dongjoon-hyun Thanks for the review. Yeah I should've changed L145 as well, from `# Run on a Kubernetes cluster in cluster deploy mode` to `# Run on a Kubernetes cluster`. I was trying to mimicking L117. However I won't do that anymore, because:

> you cannot add # after \ in bash

You're right, `#` can come after `\`. Then, L122 should also be fixed. I will revert this line, and also fix L122. Thanks.